includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -3296,6 +3296,10 @@ class AMP_Allowed_Tags_Generated {
 					),
 				),
 				'tag_spec' => array(
+					'mandatory_oneof' => array(
+						'src',
+						'srcdoc',
+					),
```

Comment:
Hm, this looks to be the same as the `mandatory_oneof` spec below:

```
'mandatory_oneof' => array(
	'src',
	'srcdoc',
),

Reply:
Oh, that's bizzare. How is Python generating an array with duplicate keys?

```php
				'tag_spec' => array(
					'mandatory_oneof' => array(
						'src',
						'srcdoc',
					),
					'amp_layout' => array(
						'supported_layouts' => array(
							6,
							2,
							3,
							7,
							9,
							1,
							4,
						),
					),
					'mandatory_oneof' => array(
						'src',
						'srcdoc',
					),
					'requires_extension' => array(
						'amp-iframe',
					),
				),
```

Reply:
I just re-ran `./bin/amphtml-update.sh` and it removed the duplicate: 4fb49daa0.

Reply:
Yeah, that's a little strange.